keel-core/src/main/kotlin/com/netflix/spinnaker/keel/core/api/ResourceSummary.kt

```diff
@@ -63,7 +64,6 @@ data class ResourceSummary(
 
 @JsonInclude(JsonInclude.Include.NON_NULL)
 data class ResourceArtifactSummary(
-  val name: String,
-  val type: ArtifactType,
-  val desiredVersion: String
```

Comment:
@erikmunson As discussed, the artifact summary under a resource will no longer contain a desired version.